codegen/testserver/singlefile/maps.graphql

```diff
@@ -30,3 +30,12 @@ scalar CustomScalar @goModel(model: "singlefile.CustomScalar")
 input NestedMapInput {
     map: MapStringInterfaceInput
 }
+
```

Comment:
Thanks for adding a specific example here so we can better avoid regressions and demonstrate correctness!